request_test.go

```diff
@@ -14,13 +14,17 @@ import (
 	"time"
 )
 
+const (
+	jsonContentType = "application/json;charset=utf-8"
```

Comment:
`application/json` is by default `utf-8`, unlike `text/html`

Reply:
i'll change it